src/analysis/souffle/examples/empty.ir

```diff
@@ -0,0 +1,4 @@
+module m0 {
```

Comment:
Add a comment as to why we need this now. Essentially, the IR is captured in the datalog files for now as we wanted to skip converting the old arcs to ir now.